File: Data Structures & Algorithms/search-insert-position/submission-6.java
class Solution {
    public int searchInsert(int[] nums, int target) {
        int l =0;
        int r = 1;
     if(target <= nums[0]){
                return 0;
            }
    if(target == nums[nums.length -1]){
        return nums.length-1;
    }
        for(int i = 0; i<nums.length-1; i++){
           
            if(nums[l] == target){
                return l;
            }
            if(nums[r]== target){
                return r;
            }
            else if(nums[l]<target && nums[r] > target){
                return r;
            }
            else{
                l++;
                r++;
            }
        }
        return nums.length;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-6.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        if len(nums) == 0:
            return 0

        nums.sort()

        count = 1
        longest = 1
        l = 0
        r = 1

        for i in range(len(nums)-1):
            if(nums[r] == nums[l] +1):
                l = l + 1
                r = r + 1
                count = count + 1
            elif(nums[r] == nums[l]):
                l = l + 1
                r = r + 1
            else:
                longest = max(longest, count)
                count = 1
                l += 1
                r += 1

        longest = max(longest, count)
        return longest